ui/pages/confirmations/components/simulation-details/simulation-details.tsx

```diff
@@ -132,6 +132,7 @@ const SimulationDetailsLayout: React.FC<{
 }> = ({ inHeader, children }) => (
   <Box
     data-testid="simulation-details-layout"
+    className="simulation-details__layout"
```

Comment:
So, I've added the `simulation-details__layout` class here, because the component is called `SimulationDetailsLayout`.

Then we have custom CSS that only applies to this new class if there is a parent class `smart-transaction-status-page`. The selector for it is: `.smart-transaction-status-page .simulation-details__layout`

<img width="213" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/80175477/c6432112-3e6e-4a80-bf42-c218600d8a89">